src/react/View.js

```diff
@@ -1,22 +1,23 @@
 import React from 'react'
```

Comment:
I would stay in "what was done and why" in commit texts.

Reply:
I would be nice to separate refactoring from the feature commits. E.g. now I could cherry-pick the refactoring commit instead of creating merge conflict by copying.

Reply:
Well... There was so much going on in this commit (that I didn't even fully understand back then) that I think it's a bit too much to ask to separate something to a different commit. For me it was just "Refactoring (and some other stuff( by pair-coding with OP" :( 

But yeah, you're right. Refactorings should be done in separate commits and they should really be refactoring (="without changing its external behavior").